src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1563,6 +1563,7 @@
 
   <Node Name="BoundDeclarationPattern" Base="BoundPattern">
     <Field Name="LocalSymbol" Type="LocalSymbol" Null="disallow"/>
```

Comment:
Having said that, like other declarations, it isn't required to be used by the generated/lowered code.

---
In reply to: [77397212](https://github.com/dotnet/roslyn/pull/13535#discussion_r77397212) [](ancestors = 77397212,77385499)